DockerFile

```diff
@@ -0,0 +1,17 @@
+FROM node:8-alpine
+
+WORKDIR /opt/datamanager
+
+RUN apk --no-cache add gcc g++ bash musl-dev make python
+
+COPY package.json ./package.json
+
+RUN npm install
+
+RUN apk --no-cache add gcc g++ bash musl-dev make python
+
+COPY . .
```

Comment:
Particularly I prefer to add the src folder specifically, not the whole root directory